code/datums/emergency_calls/emergency_call.dm

```diff
@@ -0,0 +1,217 @@
+//This file deals with distress beacons. It randomizes between a number of different types when activated.
+//There's also an admin commmand which lets you set one to your liking.
+
+
+//The distress call parent.
+/datum/emergency_call
+	var/name = ""
+	var/mob_max = 10
+	var/mob_min = 1
+	var/landmark_tag
+	var/dispatch_message = "An encrypted signal has been received from a nearby vessel. Stand by." //Message displayed to marines once the signal is finalized.
+	var/objectives = "" //Objectives to display to the members.
+	var/probability = 0 //So we can give different ERTs a different probability.
+	var/list/datum/mind/members = list() //Currently-joined members.
+	var/list/datum/mind/candidates = list() //Potential candidates for enlisting.
+	var/mob/living/carbon/leader = null
+	var/candidate_timer
+	var/cooldown_timer
+	var/spawn_type = /mob/living/carbon/human
+	var/list/specials_outfits = list()
```

Comment:
this name is grammatically strange, what is this list for?

Reply:
> this name is grammatically strange, what is this list for?

This name for specials members of team ERT. Example medic, engi and etc.